boost-maven/boost-maven-plugin/src/it/test-jaxrs-2.0/src/test/java/it/LibertyFeatureVersionIT.java

```diff
@@ -56,4 +122,41 @@ public void testLibertyFeatureVersion() throws Exception {
 
         assertTrue("The " + JAXRS_20_FEATURE + " feature was not found in the server configuration", found);
     }
+
+    @Test
+    public void testLibertyRuntimeVersion() throws Exception {
+
+        boolean found = false;
+
+        if (runtimeVersion == null) {
+            // This relies on a scan of the maven repository to find the lastest version of
+            // liberty installed. Not sure what subtle issues this might cause.
+            // if there are issues matching the runtime versions then
+            // this may be the culprit and not actually a Boost issue.
+            runtimeVersion = getLatesLibertyRuntimeVersion();
+        }
+        String assertMsg = "Runtime version is null. This may be an issue with checkig for the latest runtime version from maven and may not be a boost issue";
+        assertNotNull(runtimeVersion, assertMsg);
```

Comment:
I think you got the **assertNotNull** args backwards, see [here](http://junit.sourceforge.net/javadoc/org/junit/Assert.html#assertNull(java.lang.String,%20java.lang.Object))